backend/app/usecase/repository/userurl_fake.go

```diff
@@ -6,14 +6,32 @@ var _ UserURLRelation = (*UserURLRelationFake)(nil)
 
 // UserURLRelationFake represents in memory implementation of User-URL relationship accessor.
 type UserURLRelationFake struct {
+	userURLRelations map[string]string
 }
 
 // CreateRelation creates many to many relationship between User and URL.
 func (u UserURLRelationFake) CreateRelation(user entity.User, url entity.URL) error {
+	if u.userURLRelations == nil {
+		u.userURLRelations = make(map[string]string)
+	}
```

Comment:
Let move this logic into `NewUserURLRepoFake`

Reply:
We actually want to check whether the alias is created already here for iterating through userURLRelations slice.